### PR TITLE
CI: fix dpdgraph after dune switch

### PR DESCRIPTION
**Pull request by SkySkimmer**
This does not run the test suite as I don't know how to do that.

### CODE REVIEW

File: dev/ci/ci-coq_dpdgraph.sh [hunk 1]
```diff
@@ -10,8 +10,6 @@ git_download coq_dpdgraph
 if [ "$DOWNLOAD_ONLY" ]; then exit 0; fi
 
 ( cd "${CI_BUILD_DIR}/coq_dpdgraph"
-  autoconf
-  ./configure
-  make
-  make test-suite
+  dune build @install -p coq-dpdgraph
```
Comment (SkySkimmer) [inline dev/ci/ci-coq_dpdgraph.sh:13]: Adding 
```suggestion
  dune build @install -p coq-dpdgraph @runtest 
```
like in the opam file fails so I don't know how to run the tests.

cc @ybertot @palmskog

Comment (Alizter) [reply dev/ci/ci-coq_dpdgraph.sh:13]: What is the failure?

Comment (SkySkimmer) [reply dev/ci/ci-coq_dpdgraph.sh]: have a look

**Comment by Alizter** [comment]
It looks like we weren't running the tests in the CI for dpdgraph?! @palmskog I suggest we revert the commit merging the Dune stuff in coq-dpdgraph so that we don't break the Coq CI. Since it appears the tests are no longer working in dpdgraph, I will need to spend some time working out how to fix them (which I don't have at the moment).

**Comment by ybertot** [comment]
@Alizter can you prepare a pull-request that reverts that specific change?

**Comment by palmskog** [comment]
Fine with me to rollback the commit to `master`, but could we set up a branch for Coq 8.17 that *has* the Dune changes? I would really like to have a release for 8.17 with Dune, and this could be done from such a branch. Recall that the Coq Platform release for 8.17 is imminent.

We can solve the `master` issue over time, since there are quite a few months to Coq 8.18.

**Comment by ybertot** [comment]
The question you ask is about creating a branch on the coq-dpdgraph repository?

**Comment by palmskog** [comment]
@ybertot yes, I meant creating a branch based on the Dune `master` commit, e.g., called `coq-v8.17`, as a basis for release for 8.17. I could handle that. The idea is that the issue with Coq CI is not directly a showstopper for a Platform release.

**Comment by ppedrot** [comment]
Can we get a reasonably quick fix in dpdgraph? It's a bit annoying to have spurious CI failures, and dpdgraph can be subtle at times so that we might merge PRs that could silently introduce real regressions.

**Comment by Alizter** [comment]
I haven't had a lot of free time this week, but I am taking a look now.

**Comment by SkySkimmer** [comment]
https://github.com/coq-community/coq-dpdgraph/pull/114

**Comment by Alizter** [comment]
The failure was that the Dune version in CI is too old. We were using Dune 3.3.

**Pull request closed without merging**